.rubocop.yml

```diff
@@ -1,18 +1,36 @@
-inherit_mode:
-  merge:
-    - Exclude
-
 require:
-  - standard
-  - standard-custom
-  - standard-performance
-  - rubocop-performance
   - rubocop-rspec
 
-inherit_gem:
-  standard: config/base.yml
-  standard-performance: config/base.yml
-  standard-custom: config/base.yml
+AllCops:
+  TargetRubyVersion: 3.1
+  DisplayCopNames: true
+  SuggestExtensions: false
+  NewCops: enable
```

Comment:
I could be mistaken, but I believe we tend to keep this disabled?

Reply:
I can remove it, I enabled it to see what may need fixing, it wasn't much.